src/build.jl

```diff
@@ -187,7 +187,7 @@ function build_constraints!(M::MPSGEModel)
 end
 
 
-function consumer_income(consumer)
+function consumer_income(consumer::ScalarConsumer)
```

Comment:
there are only scalar consumers now, indexed are containers of scalars. (right?)

Reply:
Yes, that is correct.